Kickstarter-iOS/Features/ThanksProjects/Controller/ThanksViewController.swift

```diff
@@ -220,7 +220,9 @@ internal final class ThanksViewController: UIViewController, UITableViewDelegate
   }
 
   fileprivate func showRatingAlert() {
-    SKStoreReviewController.requestReview()
+    if let scene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene {
+      SKStoreReviewController.requestReview(in: scene)
+    }
```

Comment:
[`requestReview(in: UIWindowScene)`](https://developer.apple.com/documentation/storekit/skstorereviewcontroller/2851536-requestreview) should now be used instead of `requestReview()`

Reply:
Yea cool, just when making these types of StackOverflow suggested changes be careful to test that the review dialog does show up, because `UIApplication.shared.connectedScenes` is a bit ambiguous. I tested though and it shows.

Reply:
Absolutely. I was unsure of this at first as well, and made sure to find other suggestions, in apple forums for example, before deciding to go with this.